sentry-ruby/lib/sentry/transport.rb

```diff
@@ -5,16 +5,24 @@ module Sentry
   class Transport
     PROTOCOL_VERSION = '7'
     USER_AGENT = "sentry-ruby/#{Sentry::VERSION}"
+    CLIENT_REPORT_INTERVAL = 60
 
     include LoggingHelper
 
+    attr_accessor :configuration
     attr_reader :logger, :rate_limits
 
     def initialize(configuration)
+      @configuration = configuration
```

Comment:
can we only store the `send_client_reports` attribute instead of the entire configuration object?